src/notifications/AnnouncementProvider/types.js

```diff
@@ -57,6 +59,7 @@ export type AnnouncementsUserSettings = {
   getDate: () => Date,
   lastSeenDevice?: DeviceModelInfo,
   platform?: string,
+  appVersion: string,
```

Comment:
please try to make it a `appVersion?: string` in order to be truly a "non impact" PR. 👍

Reply:
Did it, feel free to review again 😃